cypress/drafts/RichTextParser/Emoticons.feature

```diff
@@ -0,0 +1,15 @@
+Feature: Emoticon parsing
+
+  Scenario Outline: Emoticons strings
+      Given there is a text string of <string>
+      And the string is not preceded immediately by a non-space character
```

Comment:
I don't like this rule. What if I want to have an emoticon at the end of a sentence, or at the end of brackets.

Reply:
I think the common workaround is to add a space to trigger the emoticon replacement and then backspace and add the extra bracket, comma, etc. Although slightly irritating, I think this is preferable. Interpreting `:)` anywhere as an emoticon would, I think, result in more unintended consequences.